### ISSUE TITLE
Crash when adding module to rocket

### ISSUE THREAD

**Issue by chris9871** [Sgt-Imalas/Sgt_Imalas-Oni-Mods#33]
I've had this happen a couple of times, but it's fairly rare.  It seems to be tied to bringing up the rocket-part selector box (clicking + from the crew module in this case).  It may be related to a fairly busy/loaded game.  The rocket in the failure below is one of 2 rockets overall, and this rocket has the basics already (engine, crew module, battery, solar panel, solar nosecone).

```
NullReferenceException

Rockets_TinyYetBig.CategorizedModMenuPatches+BuildableStatesCategoryPatch.Prefix (SelectModuleSideScreen __instance, System.Collections.Generic.Dictionary`2[BuildingDef,System.Boolean]& ___moduleBuildableState, BuildingDef ___selectedModuleDef) (at <89be5a398fcd45e78ebc5db60708e0b4>:0)
(wrapper dynamic-method) SelectModuleSideScreen.SelectModuleSideScreen.UpdateBuildableStates_Patch2(SelectModuleSideScreen,object)
SelectModuleSideScreen.SetTarget (UnityEngine.GameObject new_target) (at <1726ad14e928409087f7fe566b21e0ff>:0)
RocketModuleSideScreen.ClickAddNew (System.Single scrollViewPosition, BuildingDef autoSelectDef) (at <1726ad14e928409087f7fe566b21e0ff>:0)
RocketModuleSideScreen.<OnSpawn>b__18_0 () (at <1726ad14e928409087f7fe566b21e0ff>:0)
KButton.SignalClick (KKeyCode btn) (at <fa50cdddb14a483d94f547557a674a42>:0)
KButton.OnPointerClick (UnityEngine.EventSystems.PointerEventData eventData) (at <fa50cdddb14a483d94f547557a674a42>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute (UnityEngine.EventSystems.IPointerClickHandler handler, UnityEngine.EventSystems.BaseEventData eventData) (at <0b23117e1c7442f8b1842fe2f3bf6fcb>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute[T] (UnityEngine.GameObject target, UnityEngine.EventSystems.BaseEventData eventData, UnityEngine.EventSystems.ExecuteEvents+EventFunction`1[T1] functor) (at <0b23117e1c7442f8b1842fe2f3bf6fcb>:0)
UnityEngine.EventSystems.EventSystem:Update()

Build: U44-537329-S
```

**Comment by chris9871** [comment]
Mod version v2.4.3.1 (from mod updated in-game).

**Comment by Sgt-Imalas** [comment]
known bug, will be fixed in 2.5 release

**Comment by Sgt-Imalas** [comment]
should be fixed now (reopen if it still happens pls)
